docs/configuration.rst

```diff
@@ -50,15 +50,15 @@ Network
 +------------------------------------------+------------------------------------------------------------------------------------+-------------------+------------------+---------------+
 | Option                                   | Description                                                                        | Type              | Default          | Client/Server |
 +==========================================+====================================================================================+===================+==================+===============+
-| hadoop.cache.data.bookkeeper.port        | The port on which the BookKeeper server is listening.                              | integer           | 8899             | C & S         |
+| rubix.cache.bookkeeper.server.port       | The port on which the BookKeeper server is listening.                              | integer           | 8899             | C & S         |
```

Comment:
These network-related options might make more sense to have `rubix.network` instead of `rubix.cache` as a prefix.